pkg/resources/schema_acceptance_test.go

```diff
@@ -424,3 +779,66 @@ func checkDatabaseAndSchemaDataRetentionTime(t *testing.T, schemaId sdk.Database
 		return nil
 	}
 }
+
+func TestAcc_Schema_migrateFromVersion093(t *testing.T) {
```

Comment:
-let's migrate public schema too (just to see what happens)
-let'smigrate the one with tags too
-and the one without with_managed_access
-and maybe let's check how the newly added sf params behave after update

Reply:
Done without the last one.